plugins/anomali_threatstream/plugin.spec.yaml

```diff
@@ -306,22 +306,26 @@ connection:
     title: Username
     description: Anomali ThreatStream username
     type: string
+    example: bob_smith
```

Comment:
Should be `user1` as per https://docs.rapid7.com/insightconnect/style-guide#input-examples